subscription-service/src/test/kotlin/com/egm/stellio/subscription/support/WithTimescaleContainer.kt

```diff
@@ -25,6 +25,7 @@ interface WithTimescaleContainer {
             withEnv("POSTGRES_MULTIPLE_EXTENSIONS", "postgis,timescaledb,pgcrypto")
             withExposedPorts(5432)
             setWaitStrategy(Wait.forLogMessage(".*database system is ready to accept connections.*", 2))
+            withReuse(true)
```

Comment:
don't add this modification in the commit (do it in another specific PR that will also explain how to configure it)